App/VSTS/CreateWorkItem.tsx

```diff
@@ -0,0 +1,50 @@
+/// <reference path="../../office.d.ts" />
+import * as React from 'react';
+import { Description } from './Description';
+import { Title } from './Title';
+import { Save } from './Save';
+import { WorkItemDropdown } from './WorkItemDropdown';
+import { Classification } from './Classification';
+import { Gear } from './Gear';
+
+/**
+ * Renders all components of the Create page
+ * @class { CreateWorkItem }
+ */
+
+export class CreateWorkItem extends React.Component<{}, {}> {
+
+  public isReady: boolean; // set to false
```

Comment:
Why are we using isReady? is this a temp workaround till you integrate with Miranda's work? then its fine.